mono/mini/mini-wasm-debugger.c

```diff
@@ -1346,6 +1367,26 @@ mono_debugger_tls_thread_id (DebuggerTlsData *debuggerTlsData)
 	return 1;
 }
 
+static void
+handle_exception (MonoException *exc, MonoContext *throw_ctx, MonoContext *catch_ctx, StackFrameInfo *catch_frame)
+{
+	ERROR_DECL (error);
+	DEBUG_PRINTF (1, "handle exception - %d - %p - %p - %p\n", pause_on_exc, exc, throw_ctx, catch_ctx);
+
+	if (pause_on_exc == EXCEPTION_MODE_NONE)
+		return;
+	if (pause_on_exc == EXCEPTION_MODE_UNCAUGHT && catch_ctx != NULL)
+		return;
+
+	int obj_id = get_object_id ((MonoObject *)exc);
+	DEBUG_PRINTF (2, "handle exception - calling mono_wasm_fire_exc(): %d - message - %s\n", obj_id,  mono_string_to_utf8_checked_internal (exc->message, error));
+	
+	mono_wasm_fire_exception (obj_id, mono_string_to_utf8_checked_internal (exc->message, error));
```

Comment:
Should we handle `error` here? And pass some fallback string to `mono_wasm_fire_exception` in case of that?

Reply:
done